src/main/java/org/mockito/Mockito.java

```diff
@@ -1901,6 +1901,23 @@ public class Mockito extends ArgumentMatchers {
      */
     public static final Answer<Object> RETURNS_SELF = Answers.RETURNS_SELF;
 
+    /**
+     * Creates mock object of requested class or interface.
+     * <p>
+     * See examples in javadoc for {@link Mockito} class
+     *
+     * @param reified don't pass any values to it. It's a trick to detect the class/interface you want to mock.
+     * @return mock object
```

Comment:
Maybe it could be worth to add `@since 4.9.0`?

Reply:
thanks, added `@since` to javadoc.